src/sidebar/config/test/fetch-config-test.js

```diff
@@ -277,6 +277,26 @@ describe('sidebar/config/fetch-config', () => {
           'Unable to fetch groups'
         );
       });
+
+      it('creates a merged config and also adds back the `group` value from the host config', async () => {
+        fakeHostConfig.returns({
+          requestConfigFromFrame: {
+            origin: 'https://embedder.com',
+            ancestorLevel: 2,
+          },
+          group: '1234',
+        });
+        const appConfig = { foo: 'bar', appType: 'via' };
+        fakeJsonRpc.call.resolves({ foo: 'baz' });
+        const result = await fetchConfig(appConfig, fakeWindow);
```

Comment:
Can you add a new line before the `fetchConfig` call to make the arrange/act/assert phases of this test more obvious.